.github/workflows/merge.yml

```
@@ -0,0 +1,38 @@
+      contents: write
+      pull-requests: write
+    if:
+      github.event.review.state == 'approved' &&
+      (
+        github.event.review.author_association == 'OWNER' ||
+        github.event.review.author_association == 'MEMBER' ||
+        github.event.review.author_association == 'COLLABORATOR'
+      )
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
```

yamllint

[error] wrong indentation: expected 6 but found 4

(indentation)